ZUGFeRD/InvoiceDescriptor22UBLWriter.cs

```diff
@@ -653,14 +653,18 @@ private void _writeOptionalParty(ProfileAwareXmlTextWriter writer, PartyTypes pa
                     writer.WriteEndElement();
                 }
 
-                if (this.Descriptor.PaymentMeans?.SEPAMandateReference != null)
+                if (party == this.Descriptor.Seller)
```

Comment:
can you please evalute the partyType parameter instead of comparing the objects?